src/jekyll/_plugins/wf/WFFeedPage.rb

```diff
@@ -15,7 +15,8 @@
 module Jekyll
 
   class WFFeedPage < LanguagePage
-
+    ## TODO: Handle both atom.liquid and rss.liquid
+    ## TODO: Remove support for feed.liquid
```

Comment:
We've had requests for both ATOM and RSS for updates. Their very similar but there are a few minor differences between the XML format -- the current `feed.liquid` is an ATOM feed.

A quick check of the last 100 feed requests from H5R Updates shows 68 RSS requests and 32 ATOM requests.